cpp/src/io/parquet/reader_impl.cpp

```diff
@@ -427,11 +431,6 @@ reader::impl::impl(std::size_t chunk_read_limit,
   _metadata =
     std::make_unique<aggregate_reader_metadata>(_sources, options.is_enabled_use_arrow_schema());
 
-  // Override output timestamp resolution if requested
```

Comment:
Removed this redundant `if` and directly assigned `_options.timestamp_type = options.get_timestamp_type()`